app/views/directives/bind-service/delete-binding-select-form.html

```diff
@@ -3,18 +3,18 @@ <h3 class="mar-top-none">
 </h3>
 <form name="ctrl.bindingSelection" class="mar-bottom-lg">
   <fieldset ng-disabled="ctrl.isDisabled">
-    <div ng-repeat="binding in ctrl.bindings" class="radio">
+    <div ng-repeat="binding in ctrl.bindings | orderBy: ctrl.firstAppForBindingName : binding" class="radio">
```

Comment:
Actually I see `orderBy` can be passed a function -- disregard my comment above -- but it looks like `:binding` on the end is wrong. Isn't that the reverse sort flag?